cloudnetpy/instruments/radiometrics.py

```diff
@@ -0,0 +1,115 @@
+"""Module for reading Radiometrics MP3014 microwave radiometer data."""
+from typing import Optional
+from cloudnetpy import output
+import csv
+import numpy as np
+from cloudnetpy.exceptions import ValidTimeStampError
+from cloudnetpy import CloudnetArray
+from cloudnetpy.instruments import instruments
+
+
+def radiometrics2nc(full_path: str,
+                    output_file: str,
+                    site_meta: dict,
+                    uuid: Optional[str] = None,
+                    date: Optional[str] = None) -> str:
+    """Converts Radiometrics .csv file into Cloudnet Level 1b netCDF file.
+
+    Args:
+        full_path: Input file name.
+        output_file: Output file name, e.g. 'radiometrics.nc'.
+        site_meta: Dictionary containing information about the site and instrument.
+            Required key value pairs are `name` and `altitude` (metres above mean sea level).
+        uuid: Set specific UUID for the file.
+        date: Expected date as YYYY-MM-DD of all profiles in the file.
+
+    Returns:
+        UUID of the generated file.
+
+    Examples:
+        >>> from cloudnetpy.instruments import radiometrics2nc
+        >>> site_meta = {'name': 'Soverato', 'altitude': 21}
+        >>> radiometrics2nc('radiometrics.csv', 'radiometrics.nc', site_meta)
+
+    """
+    radiometrics = Radiometrics(full_path, site_meta)
+    radiometrics.read_raw_data()
+    radiometrics.read_lwp()
+    radiometrics.read_timestamps()
+    radiometrics.screen_time(date)
+    radiometrics.data_to_cloudnet_arrays()
+    radiometrics.add_meta()
+    attributes = output.add_time_attribute({}, radiometrics.date)
+    output.update_attributes(radiometrics.data, attributes)
+    uuid = output.save_level1b(radiometrics, output_file, uuid)
+    return uuid
+
+
+class Radiometrics:
+
+    def __init__(self, filename: str, site_meta: dict):
+        self.filename = filename
+        self.site_meta = site_meta
+        self.raw_data = []
+        self.data = {}
+        self.date = None
+        self.instrument = instruments.RADIOMETRICS
+
+    def read_raw_data(self):
+        """Reads radiometrics raw data."""
+        with open(self.filename, mode='r') as infile:
+            reader = csv.reader(infile)
+            for x in reader:
+                self.raw_data.append(x)
+        self.raw_data = self.raw_data[1:]  # First row is header
+
+    def read_lwp(self):
+        """Reads LWP values."""
+        self.data['lwp'] = np.array([row[9] for row in self.raw_data], dtype=float) * 1000  # g / m2
+
+    def read_timestamps(self):
+        """Reads timestamps."""
+        fraction_hour = []
+        time = [row[1].split()[1] for row in self.raw_data]
+        for t in time:
+            hour, minute, sec = t.split(':')
+            fraction_hour.append(float(hour) + (float(minute)*60 + float(sec)) / 3600)
```

Comment:
I would write this in a more "consistent" way:

```suggestion
            fraction_hour.append(int(hour) + int(minute) / 60 + int(sec) / 3600)
```

Reply:
Agreed